prov/sockets/src/sock_progress.c

```diff
@@ -1680,7 +1674,8 @@ static int sock_pe_progress_tx_send(struct sock_pe *pe,
 		pe_entry->conn->tx_pe_entry = NULL;
 		SOCK_LOG_INFO("Send complete\n");
 		
-		if (!(pe_entry->flags & FI_REMOTE_COMPLETE)) {
+		/* TODO: make FI_TRANSMIT_COMPLETE the default */
+		if (pe_entry->flags & FI_INJECT_COMPLETE) {
```

Comment:
@jithinjosepkl @shantonu - It's not clear to me how to make this the new default.